### DISCUSSION TITLE
Add Cover Letter Generator

### DISCUSSION THREAD

**Discussion by TaherPatan** [takline/ResumeGPT#6] [Ideas]
A Cover Letter Generator can be added as well to completely eliminate the need for any further docs. Also add a limit so that user can set if they only want a Resume and a Cover letter of 1 page so that both should only be of one page and any other extra information should be trimmed. Also the resume section does not have any projects to be added as well as no links for social media such as Linkedin, Github, LeetCode, etc which is an issue.

**Comment by takline** [comment]
Thanks, all good points. I've added a projects section in [this commit](https://github.com/takline/ResumeGPT/commit/080e2874ba52043cacd92f55a1b7bf6be4a1d46d) and will add cover letter generations to future commits
